docs/network/evm-nodes.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Running an Etherlink EVM node
+---
+
+The Etherlink EVM nodes are responsible for maintaining a copy of the Etherlink context and creating and applying new blocks that process EVM transactions.
+
+## Prerequisites
+
+- Make sure you understand the interaction between different nodes as described in [Etherlink architecture](./architecture).
+- Run an Etherlink Smart Rollup node as described in [Running an Etherlink Smart Rollup node](./smart-rollup-nodes).
+- Get the Etherlink installer kernel (`installer.hex` file), which you can build yourself as described in [Building the Etherlink kernel](./smart-rollup-nodes#building-the-etherlink-kernel) or download here: [installer.hex](/files/installer.hex).
```

Comment:
This is a prerequisites if you want to start from genesis and want contains the complete history.

Reply:
Which is what you describe after so my comment is not really important haha